release_checklist.sh

```diff
@@ -0,0 +1,242 @@
+else
+  gb_mobile_path="$script_path/$gb_mobile_path"
+fi
+
+if [[ ! "$gb_mobile_path" == *gutenberg-mobile ]]; then
+    abort "Error path does not end with gutenberg-mobile"
+fi
+
+# If including an existing github issue, look for adding aztec or incoming changes checklists
+if [[ -n "$issue_number" ]]; then
+
+  if [[ -z "$include_aztec_steps" && -z "$include_incoming_changes" ]]; then
+    abort "Nothing to update, please set the -a or -u flags to include Aztec or Incomming Changes steps."
+  fi
+  pushd "$gb_mobile_path" >/dev/null
+    issue_json=$(gh issue view "$issue_number" --json 'title,url')
+    issue_title=$(jq '.title' <<< "$issue_json")
+    issue_url=$(jq '.url' <<< "$issue_json")
+
+
+    confirm_message="Ready to update $issue_title $issue_url ?"
+
+    # Pulling the body from the gh call above i.e. with --json 'body,title,url'
+    # strips the newlines from the body. The call below does not.
+    issue_body=$(gh issue view "$issue_number" --json 'body' --jq '.body')
+
+    issue_comment="Issue updated:"
+
+    if [[ -n "$include_aztec_steps" ]]; then
+      aztec_checklist_template=$(<"$script_path/templates/release_checklist_update_aztec.md")
+      aztec_checklist_template=${aztec_checklist_template//$'\n'/\\n}
+      issue_body=$(sed -e "s/.*optional_aztec_release.*/$aztec_checklist_template/" <<< "$issue_body")
+      issue_comment="$issue_comment"$'\n\n- Added Aztec Release Checklist'
+      confirm_message="$confirm_message"$'\n\n- Add Aztec Release Checklist'
+    fi
+
+    if [[ -n "$include_incoming_changes" ]]; then
+      incoming_change_note=$checklist_message
+      if [[ -n "$incoming_change_note" ]]; then
+        incoming_change_note="**Update:** $checklist_message"
+      fi
+      incoming_changes_template=$(sed -e "s/{{incoming_change_note}}/${incoming_change_note}/g" "$script_path/templates/release_checklist_update_incoming.md")
+      incoming_changes_template=${incoming_changes_template//$'\n'/\\n}
+
+      issue_body=$(sed -e "s/.*optional_incoming_changes.*/${incoming_changes_template}/" <<< "$issue_body")
+      issue_comment="$issue_comment"$'\n\n- Added Incoming Changes Checklist'
+      confirm_message="$confirm_message"$'\n\n- Add Incoming Changes Checklist'
+    fi
+
+    if [[ -n "$checklist_message" ]]; then
+      issue_comment="$issue_comment"$'\n\n'"$checklist_message"
+    fi
+
+    if [[ -n "$debug_template" ]]; then
+      echo "$issue_body"
+      exit 0
+    fi
+
+    if [[ -z "$auto_confirm" ]]; then
+      echo ""
+      confirm_to_proceed "$confirm_message"$'\n\n'
+    fi
+
+    gh issue comment "$issue_number" --body "$issue_comment">/dev/null
+    gh issue edit "$issue_number" --body "$issue_body"
+  popd >/dev/null
+
+  exit 0;
+fi
+
+
+
+if [[ -z "$release_type" ]]; then
+  default_release_type="scheduled"
+
+  read -r -p "Please enter release type: scheduled|beta|hotfix [$default_release_type]:" release_type
+  release_type=${release_type:-$default_release_type}
+  echo ""
```

Comment:
I think it would be helpful to print the release type selected, wdyt?

```suggestion
  echo "Using '$release_type' release type"
  echo ""
```